code/pizzabot.js

```diff
@@ -1,7 +1,47 @@
+
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+const calculateTotalCost = (orderQuantity) => {
+    orderTotal = (orderQuantity * pizzaPrice)
+}
+
+const calculateCookingTime = (orderQuantity) => {
+    if (orderQuantity > 0 && orderQuantity <= 2) {
+        orderTime = 10
+    } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+        orderTime = 15
+    } else if (orderQuantity >= 6) {
+        orderTime = 20
+    }
+}
+
+let validateOrderName = (orderName, orderQuantity) => {
+    event.preventDefault();
+    if (orderName === vegetarian || orderName === hawaiian || orderName === pepperoni) {
```

Comment:
At this moment if - as a user - I will type `vegetarian pizza` instead of `Vegetarian Pizza`, that value will not be validated. Maybe it's worth adding conditional like this ?

`if (orderName.toLowerCase() === vegetarian.toLowerCase() || ... )`